pkg/controller/util.go

```diff
@@ -483,6 +513,10 @@ func makeEnv(podEnvVar *importPodEnvVar, uid types.UID) []v1.EnvVar {
 			Name:  common.InsecureTLSVar,
 			Value: strconv.FormatBool(podEnvVar.insecureTLS),
 		},
+		{
```

Comment:
Don't think we need this, per the reason above about detecting the existence of the block device file in the pod.

Reply:
Personally don't mind being explicit here.